spec/models/manageiq/providers/vmware/cloud_manager/refresher_spec.rb

```diff
@@ -103,33 +142,33 @@ def assert_ems
     expect(@ems.key_pairs.size).to eq(0)
     expect(@ems.cloud_networks.count).to eq(0)
     expect(@ems.security_groups.count).to eq(0)
-    expect(@ems.vms_and_templates.size).to eq(8)
+    expect(@ems.vms_and_templates.size).to eq(4)
     expect(@ems.vms.size).to eq(3)
-    expect(@ems.miq_templates.size).to eq(5)
+    expect(@ems.miq_templates.size).to eq(1)
     expect(@ems.orchestration_stacks.size).to eq(3)
-    expect(@ems.orchestration_templates.size).to eq(4)
+    expect(@ems.orchestration_templates.size).to eq(1)
 
     expect(@ems.direct_orchestration_stacks.size).to eq(3)
   end
 
   def assert_specific_vdc
-    @vdc = ManageIQ::Providers::Vmware::CloudManager::AvailabilityZone.where(:name => "MIQDev-Default-vCD-DualSiteStorage-PAYG").first
+    @vdc = ManageIQ::Providers::Vmware::CloudManager::AvailabilityZone.where(:name => "MIQ Devel VDC").first
```

Comment:
Nitpicking, but since we're modifying all the strings anyway, could you please also use single quotes everywhere?